/*
 * Decompiled with CFR 0.152.
 */
package org.dom4j.io;

class SAXModifyException
extends RuntimeException {
    protected SAXModifyException(Throwable cause) {
        super(cause);
    }
}

